client/driver/rkt_test.go

```diff
@@ -139,7 +139,155 @@ func TestRktDriver_Start_DNS(t *testing.T) {
 	handle2.Kill()
 }
 
-func TestRktDriver_Start_Wait(t *testing.T) {
+func TestRktDriver_Start_DNS_With_SingleApp_PodManifest(t *testing.T) {
+	if !testutil.IsTravis() {
+		t.Parallel()
+	}
+	if os.Getenv("NOMAD_TEST_RKT") == "" {
+		t.Skip("skipping rkt tests")
+	}
+
+	ctestutils.RktCompatible(t)
+	task := &structs.Task{
+		Name:   "etcd",
+		Driver: "rkt",
+		Config: map[string]interface{}{
+			"pod_manifest": `{
+				"acVersion": "1.27.0",
+				"acKind": "PodManifest",
+				"apps": [
+					{
+						"name": "coreos",
+						"image": {
+							"name": "coreos.com/etcd",
+							"labels": [
+								{
+									"name":  "version",
+									"value": "v2.0.4"
+								}
+							]
+						},
+						"app": {
+							"exec": ["/etcd"],
+							"user": "0",
+							"group": "0"
+						}
+					}
+				]
+			}`,
+			"dns_servers":        []string{"8.8.8.8", "8.8.4.4"},
+			"dns_search_domains": []string{"example.com", "example.org", "example.net"},
+		},
+		LogConfig: &structs.LogConfig{
+			MaxFiles:      10,
+			MaxFileSizeMB: 10,
+		},
+		Resources: &structs.Resources{
+			MemoryMB: 128,
+			CPU:      100,
+		},
+	}
+	ctx := testDriverContexts(t, task)
+	defer ctx.AllocDir.Destroy()
+	d := NewRktDriver(ctx.DriverCtx)
+	if _, err := d.Prestart(ctx.ExecCtx, task); err != nil {
+		t.Fatalf("error in prestart: %v", err)
+	}
+	resp, err := d.Start(ctx.ExecCtx, task)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+	defer resp.Handle.Kill()
+
+	// Attempt to open
```

Comment:
open what? :)

Reply:
This comment appears to have been copied over from other tests that do something similar: https://github.com/hashicorp/nomad/blob/master/client/driver/rkt_test.go#L131 ; While I agree this comment doesn't explain what it's attempting to open, but should we try to address that here or stick with what upstream already has?

Reply:
Up to you I guess. If you found it confusing, it doesn't hurt to help the next person understand it better.